packages/tx/src/kzg/kzg.ts

```diff
@@ -0,0 +1,24 @@
+import type { Ikzg } from '../depInterfaces'
+
+function kzgNotLoaded(): never {
+  throw Error('kzg library not loaded')
+}
+
+// eslint-disable-next-line import/no-mutable-exports
+export let kzg: Ikzg = {
+  freeTrustedSetup: kzgNotLoaded,
+  loadTrustedSetup: kzgNotLoaded,
+  blobToKzgCommitment: kzgNotLoaded,
+  computeAggregateKzgProof: kzgNotLoaded,
+  verifyKzgProof: kzgNotLoaded,
+  verifyAggregateKzgProof: kzgNotLoaded,
+}
+
+/**
+ * @param kzgLib a KZG implementation (defaults to c-kzg)
+ * @param trustedSetupPath the full path (e.g. "/home/linux/trusted_setup.txt") to a kzg trusted setup text file
+ */
+export function initKZG(kzgLib: Ikzg, trustedSetupPath = __dirname + '/trusted_setup.txt') {
+  kzg = kzgLib
+  kzg.loadTrustedSetup(trustedSetupPath)
+}
```

Comment:
If I am seeing correctly the trusted setup stuff is only used/necessary for the client. Wouldn't it be more appropriate (respectively: sufficient) to move this over to the client code to not bloat the tx library?

Side thought: this would also remain forward-compatible, so we could always still add the functionality (back) in the tx library if it turns out that this is *not sufficient*.